plugins/source/stripe/resources/services/apple_pay_domains/apple_pay_domains.go

```diff
@@ -12,7 +12,7 @@ import (
 func ApplePayDomains() *schema.Table {
 	return &schema.Table{
 		Name:        "stripe_apple_pay_domains",
-		Description: `https://stripe.com/docs/api/apple_pay_domains`,
+		Description: `https://stripe.com/docs/api`,
```

Comment:
is this desc any valuable now?

Reply:
If not specified codegen will try to generate an invalid one.